modules/ppcp-settings/resources/js/Components/Screens/Onboarding/Steps/index.js

```diff
@@ -57,7 +57,9 @@ const filterSteps = ( steps, conditions ) => {
 };
 
 export const getSteps = ( flags ) => {
+	// eslint-disable-next-line react-hooks/rules-of-hooks
```

Comment:
Instead of ignoring the "rules-of-hooks" check:
- My initial idea was to pass all conditions as arguments, like `getSteps( flags, skipPaymentsScreen )`, ie keep it a plain helper function without dependencies (would also make it more testable)
- An alternative would be to change the helper method to become a real hook -- change `getSteps(flags)` to `useSteps()` or similar, then we can drop the flags argument and collect all details inside the hook

The current mix of receiving input arguments and collecting other details via hooks is a bit messy